railties/lib/rails/generators/testing/behavior.rb

```diff
@@ -65,11 +65,15 @@ def destination(path)
         # You can provide a configuration hash as second argument. This method returns the output
         # printed by the generator.
         def run_generator(args = default_arguments, config = {})
-          capture(:stdout) do
-            args += ["--skip-bundle"] unless args.include?("--no-skip-bundle") || args.include?("--dev")
-            args += ["--skip-bootsnap"] unless args.include?("--no-skip-bootsnap") || args.include?("--skip-bootsnap")
+          args += ["--skip-bundle"] unless args.include?("--no-skip-bundle") || args.include?("--dev")
+          args += ["--skip-bootsnap"] unless args.include?("--no-skip-bootsnap") || args.include?("--skip-bootsnap")
 
+          if ENV["PRINT_STDOUT"] == "true"
```

Comment:
I'm open to feedback on this name. I didn't know if prefixing it with `RAILS` was necessary.

Reply:
What do you think about re-using `RAILS_LOG_TO_STDOUT`? It's kind of verbose but I don't mind that since it's just a debugging tool and not really for end users

Reply:
I like how explicit this it. Addressed in b81c94ce277bf61e113b38fb3063798fc348b16d